src/dymaptic.GeoBlazor.Core/Components/Layers/FeatureLayer.cs

```diff
@@ -817,6 +817,11 @@ public async Task<int[]> QueryTopObjectIds(TopFeaturesQuery query, CancellationT
         return result;
     }
 
+    /// <summary>
+    /// TimeInfo provides information such as date fields that store start and end time for each feature and the fullTimeExtent for the layer.
+    /// </summary>
+    public TimeInfo? TimeInfo { get; set; }
```

Comment:
(I realize that question applies as well to other existing properties)

Reply:
I am not sure, since it can be set before the layer is created. It also seems like this could be added to the other layers at some point.
It seems like FulTimeExtent could be a private set.

this is the info from ESRI:
TimeInfo provides information such as date fields that store start and end time for each feature and the fullTimeExtent for the layer. The timeInfo property, along with its startField and endField properties, must be set at the time of layer initialization if it is being set for a CSVLayer, GeoJSONLayer or FeatureLayer initialized from client-side features. The fullTimeExtent for timeInfo is automatically calculated based on its startField and endField properties. The timeInfo parameters cannot be changed after the layer is loaded.

Default Value:null

Reply:
OK, yeah, it sounds like we might need it to be a public setter if you use client-side features, which is a thing more akin to `GraphicsLayer`. In that case, however, we might want to enforce some of the required properties in `TimeInfo` with a constructor so they aren't accidentally left blank.